package.xml

```diff
@@ -26,4 +26,6 @@
   <run_depend>nodelet</run_depend>
   <run_depend>tf</run_depend>
 
+  <test_depend>roslaunch</test_depend>
```

Comment:
@130s I'm also not sure. I think at least `roslaunch` should be defined as `run_depend` or `test_depend` because it is explicitly used  as the argument of `find_package` in `CMakeLists.txt`.